Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM umputun/baseimage:buildgo-latest as build-backend
 
 ARG COVERALLS_TOKEN
 ARG CI
+ARG GIT_BRANCH
```

Comment:
same question. Is it needed for coverals or smth else?

Reply:
For coveralls